### DISCUSSION TITLE
Memory utilization of a k8s Pod won't get reduced after executing a task

### DISCUSSION THREAD

**Discussion by BLasan** [grafana/grafana#38416] [Observability]
Hi, I’m using prometheus along with grafana to observe the resource utilization of Pods in the k8s cluster. After getting a request utilization of cpu and memory will be shown in the dashboard. But as expected cpu will be increased and become idle(reduced) gracefully. Even though it’ll take more than expected time to get reduced. (May be couple of minutes after serving the request)

Main problem is memory. I’ve deployed some simple go server (which will do automatic garbage collection), but the memory utilization doesn’t get reduced after executing the relevant task.

But using kubernetes metrics server it shows the that memory and cpu gets reduced as expected after serving a request.
![Screenshot from 2021-08-21 17-20-04](https://user-images.githubusercontent.com/43112139/130342713-57e25833-16c6-490d-93c0-65a90c32ea53.png)

**Comment by aocenas** [comment]
Sorry but discussions are used specifically for discussions of feature requests of Grafana application. This seems like a very generic question and it also seems like not really about Grafana so I am going to lock this. You could try asking in https://community.grafana.com/ but that is also more about Grafana questions while this seems to be a question about K8s.
